lti_emailer/settings/unit_test.py

```diff
@@ -1,16 +1,13 @@
-from .local import *
+from .base import *
```

Comment:
IIRC, it was used in some emails for the course site wizard to signify which environment was being used.  I think I'll leave it out of here until and unless it becomes useful.